mesheryctl/internal/cli/root/registry/error.go

```diff
@@ -19,7 +19,7 @@ func ErrUpdateRegistry(err error, path string) error {
 }
 
 func ErrGenerateModel(err error, modelName string) error {
-	return errors.New(ErrGenerateModelCode, errors.Alert, []string{fmt.Sprintf("error generating model %s", modelName)}, []string{err.Error()}, []string{"Registrant used for the model is not available", "Failed to create model directory"}, []string{"Check network connevtivity and try again.", "Ensure sufficient permissions to allow creation of model directory"})
+	return errors.New(ErrGenerateModelCode, errors.Alert, []string{fmt.Sprintf("error generating model: %s", modelName)}, []string{err.Error()}, []string{"Registrant used for the model is not supported", "Verify the source url", "Failed to create model directory"}, []string{"Ensure one of the supported registrant is used for model generation", "Ensure correct source url is provided and in correct format", "Ensure sufficient permissions to allow creation of model directory"})
```

Comment:
1. registrants - plural

2. The registrant is not performing model generation itself. That sentence needs repressing.